examples/kubevirt-disk-uploader-tekton.yaml

```diff
@@ -156,8 +156,11 @@ metadata:
   name: kubevirt-disk-uploader-task
 spec:
   params:
+  - name: EXPORT_SOURCE_KIND
+    description: The name of the export source kind
+    type: string
   - name: POD_NAME
-    description: The name of the virtual machine
+    description: The name of the export source
```

Comment:
this description does not match to the name of the param

Reply:
Thanks. As mentioned above, the key value name is similar to constants in `ownerreference.go`. I'll update description to:
```
The name of the pod
```

Reply:
You're right, did it by mistake of copy-paste.